JarrydWorland/2021_Capstone_GP2_Team
# Hub

## Description
This PR adds a new hub scene that will serve as the starting location for the player.

# Changes include
- Added DungeonEntrance (4630101)
- Added CameraFollowBehaviour (45953f9)
- Added Hub (3a367de)

## How to test
Open the game starting from the Menu scene, ensure that non of the menu navigation is broken throughout the game, and that the player cannot escape the hub bounds. On death there should be a functional "Return to Hub" button.
